lunatrace/bsl/frontend/src/pages/project/builds/BuildListItem.tsx

```diff
@@ -28,6 +30,29 @@ interface BuildListItemProps {
   onClick: (_e: unknown) => void;
 }
 
+interface FindingForCounting {
+  vulnerability: {
+    severities: Vulnerability_Severity[];
+  };
+  purl: string;
+}
+
+export function countCriticalVulnerabilities(findings: FindingForCounting[]): number {
```

Comment:
Why do we no longer need this code in common? is it no longer used by the backend to make PR comments and so-on